tests/qos/files/qos.yml

```diff
@@ -3047,7 +2996,7 @@ qos_params:
                 q4_num_of_pkts: 300
                 q5_num_of_pkts: 80
                 q6_num_of_pkts: 80
-                limit: 80
+                limit: 150
                 lossy_weight: 8
```

Comment:
Why are we keeping ecn/wrr params for 400G and not for 100G ?

Reply:
Updated in the latest qos.yml file committed.

Reply:
I am still seeing no ecn params for 100g but available for 400g. Is this intentional ?